crates/lang/src/env_access.rs

```diff
@@ -901,4 +901,68 @@ where
     pub fn caller_is_origin(self) -> bool {
         ink_env::caller_is_origin::<E>()
     }
+
+    /// Returns the code hash of the contract at the given `account` id.
+    ///
+    /// # Example
+    ///
+    /// ```
```

Comment:
```suggestion
    /// ```rust
```

Reply:
hm, why? and what about other examples?

Reply:
Just checked and if no annotation is used, the language is Rust by default for markdown and `cargo test --doc`. So you can forget about this comment.

We already use `/// ```rust` in a number of doc comments, but let's just leave it as is there.